src/mudlet-lua/lua/StringUtils.lua

```diff
@@ -78,28 +79,29 @@ end
 
 
 --- Documentation: https://wiki.mudlet.org/w/Manual:String_Functions#string.starts
-function string.starts(String, Prefix)
-  return string.sub(String, 1, string.len(Prefix)) == Prefix
+function string:starts(prefix)
+  return string.sub(self, 1, string.len(prefix)) == prefix
 end
 
 
 
 --- Documentation: https://wiki.mudlet.org/w/Manual:String_Functions#string.title
 function string:title()
-  assert(type(self) == "string", "string.title(): no word given to capitalize")
+  local strType = type(self)
+  assert(strType == "string", "string.title(str): str as string expected, got " .. strType)
```

Comment:
Could you use the C++ variant of the error msg for consistency?

Reply:
Ok. Because consistency is good. But I still don't like it. =)

Reply:
I had no idea you don't like it! Feel free to raise it for discussion.